docs/usage/peer.md

```diff
@@ -143,6 +143,12 @@ The host used to issue the *liqoctl peer in-band* command must have **concurrent
 To this end, these subcommands feature a parallel set of flags concerning Kubernetes API access to the remote cluster, in the form `--remote-<flag>` (e.g., `--remote-kubeconfig`, `--remote-context`).
 ```
 
+```{warning}
+If you are using K3s as Kubernetes distribution and your local cluster is in a LAN behind a NAT (your home network, for example), be aware that Liqo should have been installed without the `--api-server-url` flag.
```

Comment:
The problem is not strictly related to the presence of a NAT, you will experience the same issue even with a publicly exposed k8s API.